freetextresponse/freetextresponse.py

```diff
@@ -297,10 +291,15 @@ def student_view(self, context={}):
                 'other_responses': [],
             }
         )
-        template = get_template('freetextresponse_view.html')
+
+        template = self.loader.render_django_template(
+            'public/templates/freetextresponse_view.html',
```

Comment:
Wait, I didn't think the templates were in the `public` directory...

I thought `templates` was a sibling of `public`, isn't it?
Related, do we also need to update `setup.py` to ensure the `templates` directory is packaged,
similar to https://github.com/Stanford-Online/xblock-submit-and-compare/pull/25

Reply:
Oops, moving that out. Thanks @stvstnfrd.